### ISSUE TITLE
请问训练自己的数据集为什么LOSS不收敛，结果图都是全黑或者全白。

### ISSUE THREAD

**Issue by xanxuso** [meteorshowers/RCF-pytorch#43]
May I ask why the LOSS does not converge when training our own data set, the result graphs are all black or all white.

**Comment by meteorshowers** [comment]
you can change the learning rate to stable the training process.
My training is fine on my computer.

**Comment by xanxuso** [comment]
yes,I have solved it,thank you

**Comment by AishuaiYao** [comment]
> yes,I have solved it,thank you

hello， could you mind discussion this problem with me？？？
